experimental/packages/sampler-consistent/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry Consistent Sampling
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This package provides implementations of consistent samplers that propagate sampling information across a trace.
+This implements the [draft specification][probability-sampling].
```

Comment:
No longer "draft" is it? Perhaps "development" or "experimental"?  Perhaps "draft" is accurate for OTel naming policy.